Makefile

```diff
@@ -45,14 +46,15 @@ vet:
 
 lint: golangci-lint
 	$(GOLANGCI_LINT) run
-
+lint-fix: golangci-lint ## Run golangci-lint linter and perform fixes
+	$(GOLANGCI_LINT) run --fix
 # Build the docker image
 docker-build:
-	docker build . -t ${IMG}
+	docker build . -t quay.io/joelanford/helm-operator:$(VERSION)
```

Comment:
I meant for this not to be changed at all, but I suppose this is fine for now because the only tag that can be built for export is this one.